### PR TITLE
Now font family are same

### PR DESCRIPTION
**Pull request by ArihantJain123**
I change all font family and now all font family style is same.

### CODE REVIEW

File: travel_temp.html [hunk 1]
```diff
@@ -31,6 +31,9 @@
         <meta name="summary"
             content="An open source project aimed at providing free and beautiful templates to everyone for building their portfolio websites and showcase their work to the world." />
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" />
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Noto+Serif:ital@1&display=swap" rel="stylesheet">
 
         <!--For Dark/Light mode Toggle-->
         <script src="https://code.iconify.design/1/1.0.4/iconify.min.js"></script>
```
Comment (smaranjitghose) [inline travel_temp.html:39]: This is a straightforward task. You were supposed to just replace the font family. (Check Line 41 below. Not satisfied)

File: research.html [hunk 2]
```diff
@@ -31,6 +31,9 @@
     <meta name="summary" content="An open source project aimed at providing free and beautiful templates to everyone for building their portfolio websites and showcase their work to the world." />
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" />
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Noto+Serif:ital@1&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300&display=swap" rel="stylesheet">
```
Comment (smaranjitghose) [inline research.html:37]: Why is Poppins still being fetched?

File: research.html [hunk 3]
```diff
@@ -31,6 +31,9 @@
     <meta name="summary" content="An open source project aimed at providing free and beautiful templates to everyone for building their portfolio websites and showcase their work to the world." />
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" />
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```
Comment (smaranjitghose) [inline research.html:35]: Please be mindful of the indentation

File: assets/css/style.css [hunk 4]
```diff
@@ -844,7 +845,8 @@ form {
     border-bottom: 2px solid #bb86fc;
     padding: 3%;
     color: rgb(107, 105, 105);
-    font: 95% Arial, Helvetica, sans-serif;
+    font-family: 'Noto Serif', serif;
```
Comment (smaranjitghose) [inline assets/css/style.css:848]: This font was not specified anywhere and does not go well with the UI at all

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
Please follow the Contributing.MD for the PR description

**Comment by ArihantJain123** [comment]
Now I have completed the changes in head branch please you will review my changes.

**Comment by smaranjitghose** [comment]
Use the font specified in the issue description please

**Comment by ArihantJain123** [comment]
Now I have change the font family you review the changes.

File: assets/css/cursor.css [hunk 5]
```diff
@@ -2,6 +2,7 @@ html,
 html *,
 body,
 body * {
+  font-family: 'Frank Ruhl Libre', serif;
```
Comment (smaranjitghose) [inline assets/css/cursor.css:5]: Why is this required for the cursor style?

File: assets/css/research.css [hunk 6]
```diff
@@ -58,6 +58,7 @@ html {
 /*For Changing background color of body according to theme*/
 
 body {
+  font-family: 'Frank Ruhl Libre', serif;
```
Comment (smaranjitghose) [inline assets/css/research.css:61]: See the first line of the stylesheet.

**Comment by smaranjitghose** [comment]
Too many changes for a simple PR. If we had to go through all of them, we would have done it ourselves.

**Pull request closed without merging**